geode-core/src/distributedTest/java/org/apache/geode/internal/cache/tier/sockets/UpdatePropagationDUnitTest.java

```diff
@@ -305,6 +364,40 @@ private void verifyUpdates() {
     });
   }
 
+  private void doPuts(int entries) {
+    Region r1 = getCache().getRegion(SEPARATOR + REGION_NAME);
+    assertNotNull(r1);
+    for (int i = 0; i < entries; i++) {
+      try {
+        r1.put("" + i, "" + i);
+      } catch (Exception e) {
+        e.printStackTrace();
+      }
+      try {
+        Thread.sleep(1000);
+      } catch (InterruptedException e) {
+        e.printStackTrace();
```

Comment:
Please delete all catch-blocks like this and just add the exception to a `throws` clause on the method.